core/src/Kokkos_View.hpp

```diff
@@ -1708,6 +1708,36 @@ class View : public ViewTraits<DataType, Properties...> {
   }
 };
 
+// These deduction guides result in an unmanaged View
+template <typename T>
+View(T*)->View<T>;
```

Comment:
Bringing up the question again: should this be AnonymousSpace?

Reply:
Or better: Do nothing ie do not provide deduction guidelines at all